.github/workflows/continuous-integration.yml

```diff
@@ -42,16 +42,25 @@ jobs:
 
       - run: yarn run lint
 
-      - uses: paambaati/codeclimate-action@v5.0.0
+      - name: Run Coverage Report
```

Comment:
This is the easiest way to bypass uploading a coverage report/storybook for pull requests from a fork. There would be value in getting both reports but the effort required to set that up along with the potential security risk (if we added extra secrets/forget) makes this the logical choice for now IMO.

Reply:
[source](https://github.com/orgs/community/discussions/25217#discussioncomment-3246904)